src/30_Advanced/OAuth2_Login.html

```diff
@@ -0,0 +1,62 @@
+<!---{
+  "preview": "default",
+  "default": "preview"
+  }--->
+<!--
+## Introduction
+
+This sample showcases how to use [amp-access](https://www.ampproject.org/docs/reference/components/amp-form)
+to allow an OAuth2-based login flow on your AMP pages, such as Google Sign-In.
```

Comment:
Can you extend the description to list all demonstrated sign flows to make this easier to discover via search engines...